pkg/services/provisioning/dashboards/file_reader_test.go

```diff
@@ -248,6 +250,42 @@ func TestDashboardFileReader(t *testing.T) {
 				So(folderCount, ShouldEqual, 2)
 				So(dashCount, ShouldEqual, 2)
 			})
+
+			Convey("Sanity checker should collect info about duplicate uids and folder/title's", func() {
```

Comment:
You mean folders/titles?
```suggestion
			Convey("Sanity checker should collect info about duplicate UIDs and folders/titles", func() {
```